intro_to_nhst/intro_to_nhst.md

```diff
@@ -0,0 +1,488 @@
+
+### Type 1 and Type 2 errors
+
+In NHST, there are two different kinds of errors you can have:
+
+- The first is that you might get a significant result even though actually in reality the null hypothesis is correct --- when we get a $p$-value < .05, the chance of those data happening when the null is true is small (less than 5%), but it's not zero! This is called a **Type 1 error**. You can also think of this as a false positive.
+- The second kind of error is that you may get non-significant results when actually the null hypothesis really was wrong. In other words, you might fail to detect a true effect. This is called a **Type 2 error**. You can also think of this as a false negative.
+
+We can think about these possibilities in terms of a 2x2 matrix, where your results are either significant (reject) or not (fail to reject), and the real underlying truth is that the null hypothesis is either true or false. 
+
+|               | Reject           | Fail to reject   |
+| ------------- | ---------------- | ---------------- |
+| Null is true  | Type 1 error     | Correct decision |
+| Null is false | Correct decision | Type 2 error     |
+
+In the case of the asthma researcher, a significant difference ($p < .05$) in number of asthma attacks for the two exposure groups might be a correct decision (there really is a difference between groups) or it might be a Type 1 error. 
+If they found a non-significant difference between groups ($p ≥ .05$), then they would not reject the null hypothesis and that might be a correct decision or it might be a Type 2 error.
+
+### Confidence and power
+
+The probability of a Type 1 error (getting a significant result when the null hypothesis is in fact true) is called **confidence**, and it's actually the alpha critierion we set. 
+
+As a reminder, a $p$-value is the probability of the effect observed in the data (or larger effect) if the null hypothesis were indeed true. 
+And Type 1 error is when the null is true but you reject the null hypothesis, which we do whenever $p$ is less than alpha (typically .05). 
+So by definition, we're saying we'll reject true null hypotheses 5% of the time. 
+
+<div class = "behind-the-scenes">
+<b style="color: rgb(var(--color-highlight));">Behind the scenes</b><br>
+
+**Why .05?**
+
+There's no good reason alpha should be .05, it's just a convention we've settled on. 
+It could just as easily be .01 or .1, or even .5 (although that would mean we'd be accepting 50% chance of Type 1 errors, which would add a lot of noise to the published literature!). 
+
+</div>
+
+The probability of a Type 2 error (failing to reject the null hypothesis when it's really false) is represented by the letter beta. 
+Because there are only two possible outcomes when the null is false (either correctly rejecting it or having a Type 2 error), the probability of correctly rejecting the null hypothesis (getting a significant result when the null is truly false) 1-beta, and it is called **power**.
```

Comment:
```suggestion
Because there are only two possible outcomes when the null is false (either correctly rejecting it or having a Type 2 error), the probability of correctly rejecting the null hypothesis (getting a significant result when the null is truly false) is 1-beta, and it is called **power**.
```